scripts/gm/bin/lib-gm

```diff
@@ -5,9 +5,9 @@ if [ "${DEBUG:-}" = "2" ]; then
 fi
 
 version() {
-  VERSION="v0.0.9"
-  if [ "${OUTPUT:-}" == "json" ]; then
-    printf '{"status": "success", "message": "%s"}' "$VERSION"
+  VERSION="v0.1.0"
```

Comment:
0.1 is a relatively big moment! Great work with gm Greg!

Reply:
Thanks. I wanted to make it count. :)